src/ocaml/typing/env.ml

```diff
@@ -1287,23 +1287,9 @@ let reset_declaration_caches () =
   Stamped_hashtable.clear value_declarations;
```

Comment:
All the changes in this file need review

Reply:
I reviewed all changes in this file in the commit "Resolve conflicts (with type errors outstanding)", and it seems fine.